model/commitq/githubpr/github_pr_message.go

```diff
@@ -0,0 +1,90 @@
+package githubpr
+
+import (
+	"fmt"
+
+	"github.com/mongodb/grip/level"
+	"github.com/mongodb/grip/message"
+)
+
+type GithubMergeMethod string
+
+// valid Github merge methods
+const (
+	GithubMergeMethodMerge  = GithubMergeMethod("merge")
+	GithubMergeMethodSquash = GithubMergeMethod("squash")
+	GithubMergeMethodRebase = GithubMergeMethod("rebase")
```

Comment:
This is a nit, but I think you can just do 
```
const (
 <var> <type> = <value>
 <var> = <value>
)
```
And it does the right thing.

Having said that public types for enums like this have flaws, and I think it's reasonable to put a validate method to avoid weird things if someone accidentally passes a string that isn't one of the ones you have defined for this value.

Reply:
I removed this and added the check for a valid merge method to Valid()